src/site_adapters/airbnb.ts

```diff
@@ -5,9 +5,9 @@ import { createDomScrapingAdapter } from "./domScrapingBase"
 
 const rowContainerClass = "_fhph4u"
 const rowClass = "_8ssblpx"
-const titleClass = "_1jbo9b6h"
+const titleClass = "_1c2n35az"
```

Comment:
optionally, would be nice to just make this as a separate change to master since it's not really part of this PR

Reply:
I have reversed these changes and will make them on master. Airbnb uses different classes for Chrome vs. Firefox, so I'll have to accommodate for that and might as well do that on master.